ample/main.py

```diff
@@ -242,6 +242,10 @@ def ensembling(self, optd):
         elif optd['ideal_helices']:
             ample_util.ideal_helices(optd)
             logger.info("*** Using ideal helices to solve structure ***")
+            logger.warning('Ideal helices may not solve the structure, consider using -helical_ensembles instead!')
```

Comment:
Would suggest rewording this warning message, I think it's fine to highlight the helical_ensembles option but I don't think we should imply that ideal helices won't work.

Reply:
The message you put in the docs might be more suitable